src/libgap-api.h

```diff
@@ -179,6 +179,11 @@ Obj GAP_EvalString(const char * cmd);
 // defined.
 Obj GAP_ValueGlobalVariable(const char * name);
 
+// Combines gVarName and AssGVar. For a given <name> and <value>,
+// it sets <value > to the global <name>. It returns 0 on success,
+// and 1 if <name> was read-only or constant.
```

Comment:
We need to review which return values should mean which, to ensure consistency among libgap APIs, and possible the rest of the kernel. I'd like to do that, but right now I have to do some teaching.